gke-deploy/core/resource/resource_test.go

```diff
@@ -1476,7 +1476,7 @@ default                  DaemonSet                test-app-daemonset
 foobar                   Deployment               test-app                          Yes      
 default                  Namespace                foobar                            Yes      
 test-local-deploy-all    ReplicationController    test-app-replicationcontroller    Yes      
-foobar                   Service                  test-app                          Yes      http://34.74.85.152
+foobar                   Service                  test-app                          Yes      34.74.85.152
```

Comment:
Does the unit test pass?

Reply:
the file with the change is in another changelist. I checked in the file.